python-lib/dku_kube/role.py

```diff
@@ -9,9 +9,10 @@ def create_admin_binding(user_name=None, kube_config_path=None):
     env = os.environ.copy()
     if not _is_none_or_blank(kube_config_path):
         env['KUBECONFIG'] = kube_config_path
-    logging.info("Checking clusterrolebinding with KUBECONFIG=%s" % kube_config_path)
+    logging.info("Checking ClusterRoleBinding existence with KUBECONFIG=%s" % kube_config_path)
     out = subprocess.check_output(["kubectl", "get", "clusterrolebinding", "cluster-admin-binding", "--ignore-not-found"], env=env)
     if not _is_none_or_blank(out):
-        logging.info("Clusterrolebinding already exist")
+        logging.info("ClusterRoleBinding already exists")
     else:
+        logging.info("Creating ClusterRoleBinding with KUBECONFIG=%s" % kube_config_path)
```

Comment:
the kube config path is already log before, but I guess the repetition doesn't harm anyone ;)